docs/How-To/cli/README.md

```diff
@@ -59,19 +78,7 @@ In this JSON example, an auctioneer plans to create a fixed price auction that p
   "end_time": "2022-03-01T00:00:00Z"
 }
 ```
-
-Reference the description of each field:
-
-| **Field**         |  **Description**                                                                    |
-| :---------------- | :---------------------------------------------------------------------------------- |
-| start_price       | The starting price of the selling coin; it is proportional to the paying coin denom | 
-| min_bid_price     | The minimum bid price that bidders must provide                                     |
-| selling_coin      | The selling amount of coin for the auction                                          | 
-| paying_coin_denom | The paying coin denom that bidders use to bid with                                  | 
-| vesting_schedules | The vesting schedules that release the paying coins to the autioneer                | 
-| start_time        | The start time of the auction                                                       | 
-| end_time          | The end time of the auction                                                         | 
-|                   |                                                                                     |
+the auctioneer can create a fixed price auction to `1000,000,000,000denom1` coin, where the starting price is `2.0` which means that the price of `1denom1` is `2denom2`. The auction starts at `2022-02-01T00:00:00Z` and ends at `2022-03-01T00:00:00Z`.
```

Comment:
This description uses start price of 2.0 whereas the example uses 1.0. I think it is better to change one of them to follow the other.
```suggestion
An auctioneer creates a fixed price auction with `1_000,000,000,000denom1` selling coin where the start price is `2.0`. It means that the price of `1denom1` is `2denom2`. The auction starts at `2022-02-01T00:00:00Z` and ends at `2022-03-01T00:00:00Z`.
```

Reply:
I checked and modified as suggested.